CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.5.2] - 2020-02-10
+
+###Fixed
```

Comment:
@canvural Could you insert a space before _Fixed_?

Reply:
Done!

Reply:
Thanks.